disasm/disasm.cc

```diff
@@ -1203,7 +1277,7 @@ void disassembler_t::add_instructions(const isa_parser_t* isa)
   }
 
   // ext-c
-  if (isa->extension_enabled('C')) {
+  if (isa->extension_enabled('C') || isa->extension_enabled(EXT_ZCA)) {
```

Comment:
This can be simplified to only checking for Zca if you follow my advice to implicitly enable Zca when C is present.